templates/artist/artist_application.html

```diff
@@ -5,7 +5,7 @@ <h3 class="extra-padding">Artist Application</h3>
     <div class="break">
         <hr>
     </div>
-    <p>Not sure how much to raise? Use our <a href="/funding" target=" _blank">Funding Calculator</a> to find out the best options.</p>
+    <p>Not sure how much to raise? Use our <a href="{% 'funding' %}" target="_blank">Funding Calculator</a> to find out the best options.</p>
```

Comment:
`'funding'` is supposed to be the first parameter to the `url` tag:
```suggestion
    <p>Not sure how much to raise? Use our <a href="{% url 'funding' %}" target="_blank">Funding Calculator</a> to find out the best options.</p>
```